dev-packages/browser-integration-tests/suites/tracing/metrics/pageload-measure-spans/test.ts

```diff
@@ -0,0 +1,28 @@
+import { expect } from '@playwright/test';
+import type { Event } from '@sentry/types';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import { getFirstSentryEnvelopeRequest, shouldSkipTracingTest } from '../../../../utils/helpers';
+
+// Validation test for https://github.com/getsentry/sentry-javascript/issues/12281
+sentryTest('should add browser-related spans to pageload transaction', async ({ getLocalTestPath, page }) => {
+  if (shouldSkipTracingTest()) {
+    sentryTest.skip();
+  }
+
+  const url = await getLocalTestPath({ testDir: __dirname });
+
+  const eventData = await getFirstSentryEnvelopeRequest<Event>(page, url);
+  const browserSpans = eventData.spans?.filter(({ op }) => op === 'browser');
+
+  // Spans `domContentLoadedEvent`, `connect`, `cache` and `DNS` are not
+  // always inside `pageload` transaction.
+  expect(browserSpans?.length).toBeGreaterThanOrEqual(4);
+
+  const requestSpan = browserSpans!.find(({ description }) => description === 'request');
+  expect(requestSpan).toBeDefined();
+  const measureSpan = browserSpans!.find(({ op }) => op === 'measure');
+  expect(measureSpan).toBeDefined();
+
+  expect(requestSpan!.start_timestamp).toBeLessThanOrEqual(measureSpan!.start_timestamp);
```

Comment:
q: why is this not an equality check? could it ever be that this assertion passes for timestamps that are vastly different from the expectation (i.e. delta is higher than a few ms)?